src/index.html

```diff
@@ -14,6 +14,68 @@
   </head>
   <body>
     <h1 class="logo">MA</h1>
-    <script src="scripts/main.js"></script>
+    <div id="notifications"></div>
+    <script>
+      function createNotification(message, type) {
+        const notification = document.createElement('div');
+        notification.className = type;
+        notification.dataset.qa = 'notification';
+        notification.textContent = message;
+        document.getElementById('notifications').appendChild(notification);
+      }
+
+      const firstPromise = new Promise((resolve, reject) => {
+        let clicked = false;
+        document.addEventListener('click', (event) => {
+          if (event.button === 0) {
+            clicked = true;
+            resolve('First promise was resolved');
+          }
+        });
+
+        setTimeout(() => {
+          if (!clicked) {
+            reject('First promise was rejected');
+          }
+        }, 3000);
+      });
+
+      const secondPromise = new Promise((resolve) => {
+        document.addEventListener('click', (event) => {
+          if (event.button === 0 || event.button === 2) {
+            resolve('Second promise was resolved');
+          }
+        });
```

Comment:
For the secondPromise, similar to the firstPromise, the event listener is not removed after the promise is resolved. It's a good practice to remove event listeners once they are no longer needed to prevent potential memory leaks.